packages/bridging/src/providers/near-intents/const/index.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  arbitrumOne,
+  avalanche,
+  base,
+  bnb,
+  ChainInfo,
+  gnosisChain,
+  mainnet,
+  optimism,
+  polygon,
+} from '@cowprotocol/sdk-config'
+import { HOOK_DAPP_BRIDGE_PROVIDER_PREFIX } from '../../../const'
+import type { Hex } from 'viem'
+import { BridgeStatus } from '../../../types'
+
+export const NEAR_INTENTS_HOOK_DAPP_ID = `${HOOK_DAPP_BRIDGE_PROVIDER_PREFIX}/near-intents`
+
+export const NEAR_INTENTS_SUPPORTED_NETWORKS = [
+  arbitrumOne,
+  avalanche,
+  base,
+  bnb,
+  gnosisChain,
+  mainnet,
+  optimism,
+  polygon,
+]
+
+export const NEAR_INTENTS_BLOCKCHAIN_TO_COW_NETWORK: Record<string, ChainInfo> = {
+  arb: arbitrumOne,
+  avax: avalanche,
+  base,
+  bsc: bnb,
+  eth: mainnet,
+  gnosis: gnosisChain,
+  op: optimism,
+  pol: polygon,
+}
+
+export const NEAR_INTENTS_BLOCKCHAIN_TO_NATIVE_WRAPPED_TOKEN_ADDRESS: Record<string, Hex> = {
```

Comment:
would it make sense to use `WRAPPED_NATIVE_CURRENCIES`?

Reply:
Also, should you have a type for the possible values of near blockchains? this would avoid uncoherence between NEAR_INTENTS_BLOCKCHAIN_TO_COW_NETWORK, NEAR_INTENTS_BLOCKCHAIN_TO_NATIVE_WRAPPED_TOKEN_ADDRESS etc